### DISCUSSION TITLE
Different pricing determinates for the same service

### DISCUSSION THREAD

**Discussion by jltang-kp** [CMSgov/price-transparency-guide#242] [Ideas]
Edited post:

We have certain pricing determinates that would also the negotiated rate of a service. These include CPT modifiers, Patient Age, Rendering Provider's Specialty, Type of Service, Provider Organization, Medication, Provider Type, DRGs/percentage of DRGs

For example we have a contract where MDs and mid-level provider receives a different rate for an office visit 99212, however in the current CMS schema there is no way to represent the difference in price for the same service. 
Yes, in this example the description field may be leveraged however if some sort of software were to read the MRF I don't believe the description field would be valid and a new pricing determinate field maybe needed. 

I know there have been other similar pricing determinate discussions as well and really think that this needs to be addressed for the 7/1 schema in order to display our contracts along with many other payers contracts correctly in the MRF1 format.

https://github.com/CMSgov/price-transparency-guide/discussions/201
https://github.com/CMSgov/price-transparency-guide/discussions/23#discussion-3288379
https://github.com/CMSgov/price-transparency-guide/discussions/9#discussion-3255361
https://github.com/CMSgov/price-transparency-guide/discussions/12#discussion-3258068

**Comment by kavkuk** [comment]
@jltang-kp Thanks for bringing this up again.  I agree, need more discussion and guidance on this. 

CPT modifier codes need a field in the schema, and/or as an optional custom field that can be used for either modifier codes, or special pricing e.g. pricing based on Age Category (#201 ).  Leveraging the description field does not make sense.

**Reply by jltang-kp** [reply]
Right, and thinking about percent of billed, upon discussing this with more folks in our org and how the MRF will be consumed. We don't think it makes sense to place a percentage of billed in the negotiated rate field unless the file reader can understand the differences.

**Reply by kavkuk** [reply]
@shaselton With the addition of the Modifier field in the recent schema, is the assumption accurate, that the modifier field can be used to define instances of special pricing, e.g. pricing based on Age Category?

**Reply by jltang-kp** [reply]
I did ask this question during the the Jan 12th webinar and @shaselton-usds  responded. So it looks like billing codes that also ask for age, gender or other price determinates based on member demographics should not be included in the MRF1 as only base rates should be reported. Though this has proven to be a bit challenging as some of our contracts are configured with those pricing determinates only and no base rate, for those contracts we are looking to either report those rates in billing codes with price determinates unlisted or may have to list a PDF of the paper contract in a companion file. The regulation mentions alternate payment arrangements that cant be handled by the schema

**Comment by angelamarie0975** [comment]
I am in agreement that it makes sense to add modifier codes to the schema as this plays a huge part in the negotiated rates - HOWEVER, adding the modifier code will increase the file size as more data will be reported.

For percent of billed - has anyone even figured out how to calculate the percent of billed for this file? Since the requirements state to display the value as a dollar amount, this means that we have to do a calculation using a billed amount, but the problem with the MRF is that we do not have this charge to do the calculation. I have heard some people talk about trying to get an average billed amount based on the claim history, but this adds more complexity to the file data and poses other issues. If a particular contracted provider doesn't happen to have any claims history to derive a billed charge from, then how could the data be calculated in order to display that dollar amount in the MRF? Also - if the answer is to use claims history to derive a billed charge, then that means more potential security issues because we will be using claims historical data for parts of this file. I know that there were already concerns about using historical data for the allowed_amounts file, so this would just add to those concerns. Just my 2 cents worth. :)

IF we have to display percent of billed rates, then would it make sense to add some other type of attribute or a new value to the negotiated arrangement attribute specific to percentage of billed? If we can do that, then we could just display the negotiated percentage value instead of trying to determine an average billed amount based on claims history. I don't know if this is feasible, but I am having a hard time trying to figure out how to derive the percent of billed at this time.

**Reply by frenchcanuk** [reply]
That is a great question as well.  For one of our plan most practitionner bill a full amount and we then apply a discount. For those case there is really no fee schedule and we only get data if we actually get a claim from them.  

Some also use a percentage of the medicare allowed amount. Curious to see what other people are doing in this scenario

**Reply by angelamarie0975** [reply]
@shaselton-usds Bumping this discussion up again as well. Is there any possibility of adding Modifier Code or Percentage Rate for % of Billed contracts?

**Comment by williamcullenquartz** [comment]
I agree with what has been voiced previously. We also have contracts that reimburse different rates for the same service based on criteria that isn't part of the current schema (as an example: Bill Type, Amount Billed). In this situation, our contract would allow for three different rates for the same service. Based on the current schema only one rate would pull onto the MRF. Is it appropriate to only report one rate?

**Comment by rgouldkc** [comment]
We are a regional PPO network in the Kansas City metro with providers operating on both sides of the state line. In addition to the items already mentions, we have other criteria that drives a contract to different rates: State where the service is provided (usually KS or MO but there can be others), zip codes which are generally used to define a specific county or counties, and in one case a specific street address.  Haven't yet come up with an idea that works well with the current schema to handle this.
